### ISSUE TITLE
Panic during setup

### ISSUE THREAD

**Issue by oleksandr** [documize/community#64]
1. Downloaded the latest version for OSX
2. Create DB (utf8)
3. Executed: ./documize-community-darwin-amd64 -port 5000 -db='root:password@tcp(localhost:3306)/documize'

Received immediately the following output:

```
INFO[0000] Temporary upload directory: /var/folders/ql/swjcy9z513x1dgzqkbd8gjnjydr16h/T/documize/_uploads/  env=Non-production
INFO[0000] Running database checks, this may take a while...  env=Non-production
ERRO[0000] Entering database set-up mode because the database is empty.  env=Non-production error=no tables stack=goroutine 1 [running]:
github.com/documize/community/core/log.Error(0x782846, 0x3c, 0xac2f60, 0xc4201ee1a0)
	/Users/Harvey/code/go/src/github.com/documize/community/core/log/logger.go:64 +0xab
github.com/documize/community/core/database.Check(0xc4202f9b30, 0x7fff5fbff8e9, 0x2a, 0x0)
	/Users/Harvey/code/go/src/github.com/documize/community/core/database/check.go:121 +0x952
github.com/documize/community/core/api/request.init.1.func1(0xe1c1b0, 0x741afe, 0x2, 0x741afe)
	/Users/Harvey/code/go/src/github.com/documize/community/core/api/request/init.go:76 +0x1a4
github.com/documize/community/core/environment.Parse(0x741afe, 0x2)
	/Users/Harvey/code/go/src/github.com/documize/community/core/environment/environment.go:95 +0xa53
main.main()
	/Users/Harvey/code/go/src/github.com/documize/community/cmd/documize-community/documize.go:26 +0x36

INFO[0000] database.Check(Db) !OK, going into setup mode  env=Non-production
INFO[0000] Non-production environment logging enabled
INFO[0000] Please set DOCUMIZESALT or use -salt with this value: 1VfKc7m53121N869b  env=Non-production
INFO[0000] Documize registered 9 sections and tabs       env=Non-production
INFO[0000] Documize API token is not valid: Documize token is empty  env=Non-production
INFO[0000] Starting Community Edition version 0.34.0     env=Non-production
INFO[0000] Serving SETUP web app                         env=Non-production
INFO[0000] Starting non-SSL server on 5000               env=Non-production
[negroni] listening on :5000
```

Opened http://localhost:5000, got redirected to /setup, filled the form and clicked setup. As a result:

```
INFO[0070] Database checks: will execute the following update files: db_00000.sql, db_00001.sql, db_00002.sql, db_00003.sql, db_00004.sql, db_00005.sql, db_00006.sql, db_00007.sql, db_00008.sql  env=Non-production
INFO[0070] Processing migration file: db_00000.sql       env=Non-production
INFO[0077] Processing migration file: db_00001.sql       env=Non-production
INFO[0077] Processing migration file: db_00002.sql       env=Non-production
INFO[0077] Processing migration file: db_00003.sql       env=Non-production
ERRO[0077] Database checks: failed:                      env=Non-production error=Error 1060: Duplicate column name 'global' stack=goroutine 34 [running]:
github.com/documize/community/core/log.Error(0x768a32, 0x19, 0xac3160, 0xc4202ac6e0)
	/Users/Harvey/code/go/src/github.com/documize/community/core/log/logger.go:64 +0xab
github.com/documize/community/core/database.migrateEnd(0xc42028d440, 0xac3160, 0xc4202ac6e0, 0xc42028d401, 0x0, 0x0)
	/Users/Harvey/code/go/src/github.com/documize/community/core/database/migrate.go:163 +0xc5
github.com/documize/community/core/database.Migrate(0xecfca0000, 0xc431de46a4, 0xe1d700)
	/Users/Harvey/code/go/src/github.com/documize/community/core/database/migrate.go:229 +0x352
github.com/documize/community/core/database.Create(0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/core/database/create.go:119 +0x576
net/http.HandlerFunc.ServeHTTP(0x800338, 0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/usr/local/go/src/net/http/server.go:1726 +0x44
github.com/documize/community/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc420282d70, 0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/gorilla/mux/mux.go:98 +0x255
github.com/documize/community/vendor/github.com/codegangsta/negroni.Wrap.func1(0x11257d8, 0xc420137a40, 0xc420dc6ff0, 0xc420287c60)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:41 +0x4d
github.com/documize/community/vendor/github.com/codegangsta/negroni.HandlerFunc.ServeHTTP(0xc420257fe0, 0x11257d8, 0xc420137a40, 0xc420dc6ff0, 0xc420287c60)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:24 +0x4e
github.com/documize/community/vendor/github.com/codegangsta/negroni.middleware.ServeHTTP(0xac4e20, 0xc420257fe0, 0xc420246040, 0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:33 +0xa7
github.com/documize/community/vendor/github.com/codegangsta/negroni.(middleware).ServeHTTP-fm(0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:33 +0x60
github.com/documize/community/core/api/endpoint.Authorize(0x11257d8, 0xc420137a40, 0xc420dc6ff0, 0xc420287c40)
	/Users/Harvey/code/go/src/github.com/documize/community/core/api/endpoint/authentication_endpoint.go:226 +0xcc
github.com/documize/community/vendor/github.com/codegangsta/negroni.HandlerFunc.ServeHTTP(0x800068, 0x11257d8, 0xc420137a40, 0xc420dc6ff0, 0xc420287c40)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:24 +0x4e
github.com/documize/community/vendor/github.com/codegangsta/negroni.middleware.ServeHTTP(0xac4e20, 0x800068, 0xc420246020, 0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:33 +0xa7
github.com/documize/community/vendor/github.com/codegangsta/negroni.(*Negroni).ServeHTTP(0xc42028d590, 0x11257d8, 0xc420137a00, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:73 +0xe4
github.com/documize/community/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc420076f50, 0x11257d8, 0xc420137a00, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/gorilla/mux/mux.go:98 +0x255
github.com/documize/community/vendor/github.com/codegangsta/negroni.Wrap.func1(0x11257d8, 0xc420137a00, 0xc420dc6ff0, 0xc420287c00)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:41 +0x4d
github.com/documize/community/vendor/github.com/codegangsta/negroni.HandlerFunc.ServeHTTP(0xc42009c220, 0x11257d8, 0xc420137a00, 0xc420dc6ff0, 0xc420287c00)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:24 +0x4e
github.com/documize/community/vendor/github.com/codegangsta/negroni.middleware.ServeHTTP(0xac4e20, 0xc42009c220, 0xc4202f4000, 0x11257
ERRO[0077] database.Create()                             env=Non-production error=Error 1060: Duplicate column name 'global' stack=goroutine 34 [running]:
github.com/documize/community/core/log.Error(0x7616ef, 0x11, 0xac3160, 0xc4202ac6e0)
	/Users/Harvey/code/go/src/github.com/documize/community/core/log/logger.go:64 +0xab
github.com/documize/community/core/database.Create(0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/core/database/create.go:120 +0x5a8
net/http.HandlerFunc.ServeHTTP(0x800338, 0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/usr/local/go/src/net/http/server.go:1726 +0x44
github.com/documize/community/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc420282d70, 0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/gorilla/mux/mux.go:98 +0x255
github.com/documize/community/vendor/github.com/codegangsta/negroni.Wrap.func1(0x11257d8, 0xc420137a40, 0xc420dc6ff0, 0xc420287c60)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:41 +0x4d
github.com/documize/community/vendor/github.com/codegangsta/negroni.HandlerFunc.ServeHTTP(0xc420257fe0, 0x11257d8, 0xc420137a40, 0xc420dc6ff0, 0xc420287c60)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:24 +0x4e
github.com/documize/community/vendor/github.com/codegangsta/negroni.middleware.ServeHTTP(0xac4e20, 0xc420257fe0, 0xc420246040, 0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:33 +0xa7
github.com/documize/community/vendor/github.com/codegangsta/negroni.(middleware).ServeHTTP-fm(0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:33 +0x60
github.com/documize/community/core/api/endpoint.Authorize(0x11257d8, 0xc420137a40, 0xc420dc6ff0, 0xc420287c40)
	/Users/Harvey/code/go/src/github.com/documize/community/core/api/endpoint/authentication_endpoint.go:226 +0xcc
github.com/documize/community/vendor/github.com/codegangsta/negroni.HandlerFunc.ServeHTTP(0x800068, 0x11257d8, 0xc420137a40, 0xc420dc6ff0, 0xc420287c40)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:24 +0x4e
github.com/documize/community/vendor/github.com/codegangsta/negroni.middleware.ServeHTTP(0xac4e20, 0x800068, 0xc420246020, 0x11257d8, 0xc420137a40, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:33 +0xa7
github.com/documize/community/vendor/github.com/codegangsta/negroni.(*Negroni).ServeHTTP(0xc42028d590, 0x11257d8, 0xc420137a00, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:73 +0xe4
github.com/documize/community/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc420076f50, 0x11257d8, 0xc420137a00, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/gorilla/mux/mux.go:98 +0x255
github.com/documize/community/vendor/github.com/codegangsta/negroni.Wrap.func1(0x11257d8, 0xc420137a00, 0xc420dc6ff0, 0xc420287c00)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:41 +0x4d
github.com/documize/community/vendor/github.com/codegangsta/negroni.HandlerFunc.ServeHTTP(0xc42009c220, 0x11257d8, 0xc420137a00, 0xc420dc6ff0, 0xc420287c00)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:24 +0x4e
github.com/documize/community/vendor/github.com/codegangsta/negroni.middleware.ServeHTTP(0xac4e20, 0xc42009c220, 0xc4202f4000, 0x11257d8, 0xc420137a00, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:33 +0xa7
github.com/documize/community/vendor/github.com/codegangsta/negroni.(middleware).ServeHTTP-fm(0x11257d8, 0xc420137a00, 0xc420dc6ff0)
	/Users/Harvey/code/go/src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:33 +0
```

**Comment by HarveyKandola** [comment]
Looking into it and will get back to you...

**Comment by HarveyKandola** [comment]
Fixed in v0.34.1 release so please download and try it!
